test/byContract/IexecPocoBoost/IexecPocoBoost.test.ts

```diff
@@ -873,7 +872,7 @@ describe('IexecPocoBoost', function () {
             expect(deal2.botFirst).to.be.equal(3); // next index after last task of deal1:{0, 1, 2}
             expect(deal2.botSize).to.be.equal(5);
             // Verify request is fully consumed
-            await expectOrderConsumed(iexecPocoBoostInstance, requestOrderHash, 8);
+            await expectOrderConsumed(requestOrderHash, 8);
```

Comment:
We can put this in a shared constant:
```suggestion
            await expectOrderConsumed(requestOrderHash, requestOrderVolume);
```

Reply:
Thanks [Declare requestOrderVolume once](https://github.com/iExecBlockchainComputing/PoCo/pull/153/commits/482b2d2991553092b487b0776fc443bc2b651faa) :heavy_check_mark: